sdks/js/packages/core/react/components/organization.tsx

```diff
@@ -49,9 +49,11 @@ export const CreateOrganization = ({
   const { client } = useFrontier();
 
   async function onSubmit(data: any) {
+    if (!client) return;
+
     const {
       data: { organization }
-    } = await client.createOrganisation(data);
+    } = await client.frontierServiceCreateOrganization(data);
```

Comment:
@pyadav is it possible to remove frontService prefix from this? That seems redundant.